vendor/Wren/wren.odin

```diff
@@ -0,0 +1,203 @@
+package wren
+
+// TODO: import on Linux
+when ODIN_OS == "windows" do foreign import "lib/wren.lib"
+
+import c "core:c"
+
+WREN_VERSION_MAJOR :: 0;
+WREN_VERSION_MINOR :: 4;
+WREN_VERSION_PATCH :: 0;
+WREN_VERSION_STRING :: "0.4.0";
+WREN_VERSION_NUMBER :: 4000;
+
+ReallocateFn         :: #type proc "c" (memory: rawptr, newSize: u64, userData: rawptr) -> rawptr;
+ForeignMethodFn      :: #type proc "c" (vm: ^VM);
+FinalizerFn          :: #type proc "c" (data: rawptr);
+ResolveModuleFn      :: #type proc "c" (vm: VM, importer: cstring, name: cstring) -> cstring;
+LoadModuleCompleteFn :: #type proc "c" (vm: ^VM, name: cstring, result: LoadModuleResult);
+LoadModuleFn         :: #type proc "c" (vm: ^VM, name: cstring) -> LoadModuleResult;
+BindForeignMethodFn  :: #type proc "c" (vm: ^VM, module: cstring, className: cstring, isStatic: bool, signature: cstring) -> ForeignMethodFn;
+WriteFn              :: #type proc "c" (vm: ^VM, text: cstring);
+ErrorFn              :: #type proc "c" (vm: ^VM, type: ErrorType, module: cstring, line: c.int, message: cstring);
+BindForeignClassFn   :: #type proc "c" (vm: ^VM, module: cstring, className: cstring) -> ForeignClassMethods;
+
+ErrorType :: enum i32 {
+    WREN_ERROR_COMPILE,
+    WREN_ERROR_RUNTIME,
+    WREN_ERROR_STACK_TRACE,
+}
+
+InterpretResult :: enum i32 {
+    WREN_RESULT_SUCCESS,
+    WREN_RESULT_COMPILE_ERROR,
+    WREN_RESULT_RUNTIME_ERROR,
+}
+
+Type :: enum i32 {
+    WREN_TYPE_BOOL,
+    WREN_TYPE_NUM,
+    WREN_TYPE_FOREIGN,
+    WREN_TYPE_LIST,
+    WREN_TYPE_MAP,
+    WREN_TYPE_NULL,
+    WREN_TYPE_STRING,
+    WREN_TYPE_UNKNOWN,
+}
+
+VM :: struct {}
+
+Handle :: struct {}
+
+LoadModuleResult :: struct {
+    source : cstring,
+    onComplete : LoadModuleCompleteFn,
+    userData : rawptr,
+}
+
+ForeignClassMethods :: struct {
+    allocate : ForeignMethodFn,
+    finalize : FinalizerFn,
+}
+
+Configuration :: struct {
+    reallocateFn : ReallocateFn,
+    resolveModuleFn : ResolveModuleFn,
+    loadModuleFn : LoadModuleFn,
+    bindForeignMethodFn : BindForeignMethodFn,
+    bindForeignClassFn : BindForeignClassFn,
+    writeFn : WriteFn,
+    errorFn : ErrorFn,
+    initialHeapSize : c.size_t,
+    minHeapSize : c.size_t,
+    heapGrowthPercent : c.int,
+    userData : rawptr,
+}
+
+@(default_calling_convention="c")
```

Comment:
You could use `@(default_calling_convention="c", link_prefix="wren")`

That way you don't have to provide the link name for each procedure.

```odin
 @(link_name="wrenInitConfiguration")
    InitConfiguration :: proc(configuration : ^Configuration) ---;
```
Would turn into:
```odin
InitConfiguration :: proc(configuration : ^Configuration) ---;
```